Makefile

```diff
@@ -99,12 +99,15 @@ doc-open:
 size:
 	find . -name "esp-build.map" -exec ${IDF_PATH}/tools/idf_size.py {} \;
 
-build-esp32-bin:
-	cargo +esp espflash save-image --package micro-rdk-server --merge --chip esp32 target/xtensa-esp32-espidf/micro-rdk-server-esp32.bin -T micro-rdk-server/esp32/partitions.csv -s 4mb  --bin micro-rdk-server-esp32 --target=xtensa-esp32-espidf  -Zbuild-std=std,panic_abort --release
+build-esp32-bin: build-esp32-ota
```

Comment:
only annoying thing is you'll see the same terminal output twice, since the output doesn't differentiate between a full build or app image build.

Reply:
I spent some time seeing if there was a way to do the merge as its own step, or to request production of both merged and unmerged simultaneously, but it yielded nothing.